SKILL.tsv

```diff
@@ -5291,20 +5291,20 @@ SKILL_20151001_005290	$Corsair 2nd Circle required
 SKILL_20151001_005291	$Lv. 2 Create Shoggoth required
 SKILL_20151016_005292	$Critical Chance: +#{CaptionRatio}#{nl}Block: +#{SkillFactor}#% {nl}Evasion: -#{CaptionRatio2}#{nl}Duration: #{CaptionTime}# seconds
 SKILL_20151016_005293	$Spear Lunge
-SKILL_20151016_005294	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Deals damages on an enemy infront by thrusting and makes it vulnerable to Thrust attacks.
+SKILL_20151016_005294	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Stab the enemy in front of you, and make it more vulnerable to [Pierce] attacks.
 SKILL_20151016_005295	ATK #{SkillAtkAdd}#{nl}Attribute ATK #{CaptionRatio}#%{nl}Duration 6 Secs
 SKILL_20151016_005296	Special Formation
 SKILL_20151016_005297	Creates a formation which brings squad members together. The skill cooldown time of the squad members that entered into the formation will be reduced in half. Duration will increase by Skill Level, and the number of members in the squad.
 SKILL_20151016_005298	Cooldown time reduced by 50%{nl}Duration at minimum #{CaptionRatio}#Secs ~ Max #{CaptionRatio2}#Secs
 SKILL_20151016_005299	Summon Carriage
-SKILL_20151016_005300	{#993399}{ol}[Magic]{/}{/}{nl}Creates a gravity field that is a line shaped and pull the nearby enemies to attack.
-SKILL_20151016_005301	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Deals damages on an enemy by throwing Ice Sphere.
+SKILL_20151016_005300	{#993399}{ol}[Magic]{/}{/}{nl}Creates a gravity field in front of you in a line that pulls nearby enemies into it.
+SKILL_20151016_005301	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Damages an enemy by throwing an Ice Sphere.
 SKILL_20151016_005302	$Cannon Shot
 SKILL_20151016_005303	$Shootdown
 SKILL_20151016_005304	$Siege Burst
 SKILL_20151016_005305	$Cannon Blast
-SKILL_20151016_005306	{#993399}{ol}[Magic]{/}{/}{nl}Makes a owl sculpture of the protection which attacks the enemy at the front.
-SKILL_20151016_005307	Neutralilzes all magics around you.
+SKILL_20151016_005306 	{#993399}{ol}[Magic]{/}{/}{nl}Makes a owl sculpture of the protection which attacks enemies in front of it.
```

Comment:
Carve an Owl Sculpture to protect yourself from the attacks of enemies in front of it.